build_test.go

```diff
@@ -103,14 +99,15 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 		buffer = bytes.NewBuffer(nil)
 		logEmitter := nodeengine.NewLogEmitter(buffer)
 
-		build = nodeengine.Build(entryResolver, dependencyManager, environment, logEmitter, clock)
+		build = nodeengine.Build(entryResolver, dependencyManager, environment, sbomGenerator, logEmitter, clock)
 
 		buildContext = packit.BuildContext{
 			CNBPath: cnbDir,
 			Stack:   "some-stack",
 			BuildpackInfo: packit.BuildpackInfo{
-				Name:    "Some Buildpack",
-				Version: "1.2.3",
+				Name:        "Some Buildpack",
+				Version:     "1.2.3",
+				SBOMFormats: []string{"application/vnd.cyclonedx+json", "application/spdx+json"}, // TODO: can we replace with concrete types?
```

Comment:
These are constants available in the `sbom` package: https://github.com/paketo-buildpacks/packit/blob/v2/sbom/formats.go#L4-L6